kivymd/uix/snackbar.py

```diff
@@ -372,6 +358,38 @@ class Snackbar(MDBoxLayout):
     and defaults to `'[5, 5, 5, 5]'`
     """
 
+    snackbar_animation_dir = OptionProperty(
+        "Bottom",
+        options=[
+            "Top",
+            "Bottom",
+            "Left",
+            "Right",
+        ],
+    )
+    """
+    Snackbar animation direction. Available options are: (`"Top"`, `"Bottom"`, `"Left"`, `"Right"`).
+
+    :attr:`snackbar_animation_dir` is an :class:`~kivy.properties.OptionProperty`
+    and defaults to `'Bottom'`.
```

Comment:
Use PEP8

Reply:
Sounds good to me! I am just getting used to be pedantic about code formatting. I will make the adjustments later today and update the PR! Thank you for the review!